index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href="styles.css" rel="stylesheet"> 
+</head>
+<body>
+    <header>
+        <a href="index.html"><img src="media/Logo.svg" alt="Logo" class="logo"></a>
+        <nav>
+            <a href="index.html"><img src="media/Union.svg" alt="menu icon" class="hamburger"></a>
+        </nav>
+    </header>
+    <section>
+        <h1>I'm Chime Glad to see you!</h1>
+        <P>
+            I'm a software developer and I can help you build a product,
+            feature or website, look through my work and experience! If
+            you like what you see and have a project you need coded don't
+            hesitate to contact me.  
+        </P>
+        <br>
+        <a href="#">LET'S CONNECT</a>
+        <br>
+        <br>
+        <div>
+            <img src="media/twitter.svg" alt="twitter" class="icons">
+            <img src="media/Linkedin.svg" alt="Linkedin" class="icons">
+            <img src="media/M.svg" alt="icon" class="icons">
+            <img src="media/github.svg" alt="github" class="icons">
+            <img src="media/hand.svg" alt="icon" class="icons">
+        </div>
```

Comment:
- [ ] Please consider using a `ul` tag to group social media links, and `li` per icon; each icon should have a link (a tag), eg.:
```
<ul>
  <li>
    <a href="/"><img src="media/twitter.svg" alt="twitter" class="icons"></a>
  </li>
</ul>
```